ipaserver/plugins/krbtpolicy.py

```diff
@@ -311,9 +314,26 @@ def execute(self, uid=None, **options):
                 def_values[a] = None
         # if reseting global policy - set values to default
         else:
-            def_values = _default_values
+            def_values = deepcopy(_default_values)
```

Comment:
The default values dict contains just strings and ints. ``_default_values.copy()`` does the trick, too.

Reply:
updated, thanks.